app.py

```diff
@@ -80,7 +80,7 @@ def compute_sample_sizes(round_contest):
     sampler = get_sampler(election)
 
     # format the options properly
-    raw_sample_size_options = sampler.get_sample_sizes(sample_results(election))[election.contests[0].id]
+    raw_sample_size_options = sampler.get_sample_sizes(sample_results(election))[round_contest.contest_id]
```

Comment:
For some reason `bgcompute` iterates over contests and calls `compute_sample_sizes` here. 
This is kind of redundant: `get_sample_sizes` has all the info we need for a round, could easily refactor this method and `bgcompute` to be a lot more simple.